scss/_tables.scss

```diff
@@ -55,6 +55,13 @@
     width: $spacer * 2;
     max-width: $spacer * 2;
   }
+
+  // Use negative margins on icons in tables to preserve row height
+  svg,
+  img {
+    margin-top: $table-cell-icon-margin-top;
```

Comment:
First, I'd add a `display: block;` to ensure the correct display of those images. Then, I think I'd made only 1 variable here `$table-icon-margin-y (: -.9375rem)`. It seems to fit well with the design + It would handle well larger icons.

Reply:
You cannot add a `display: block;`, because user can add some content next to icon or thumbnail, which is the case in our example !
For the margin, I chose these values after some tests, and having the same at top and bottom does not fit with design: 
Small table with -.625rem:
![image](https://user-images.githubusercontent.com/16681352/207902844-d8795dcf-d56e-48d1-ba33-8ce290c53918.png)
Small table with -.563rem and -.313rem:
![image](https://user-images.githubusercontent.com/16681352/207903007-b2c41c15-1ff1-49a7-b2d1-6c2a05dc9837.png)

You can see that the icon is not i the middle of the line with -.625rem...

Reply:
Oh you're right, I made my tests on a single image ! Tables behave weird (or compute weird) with inline elements (or maybe the invert, inline element have a weird behavior inside tables). I don't know any other workaround apart this display tweak for this issue. Any thought on this ?

FYI, with same margin:
![image](https://user-images.githubusercontent.com/91960143/207927035-f8731d85-d3a6-4e96-b556-c2c45778719c.png)

with actual margin:
![image](https://user-images.githubusercontent.com/91960143/207928499-0d1c1d9c-ebba-4bfb-90ef-94eb6291409b.png)

with `.d-block`:
![image](https://user-images.githubusercontent.com/91960143/207927151-9d1efe18-b0e6-4756-b43a-235343e96475.png)

Actual behavior:
![image](https://user-images.githubusercontent.com/91960143/207928972-2348f7ae-b39b-4c10-ba96-49833b112b51.png)

Reply:
Fine as-is for now, if people limit their icon size.